scripts/build/clean.js

```diff
@@ -0,0 +1,45 @@
+
+const {exec} = require('child_process');
+const fs = require('fs');
+
+const {sync: globSync} = require('glob');
+
+function main() {
+  removeDirectory('build');
+  removeDirectory('.rewrite-tmp');
+  removeFilesOfType('js');
+  removeFilesOfType('d.ts');
+  removeFilesOfType('map');
+}
+
+function removeDirectory(directory) {
+  exec(`rm -rf ${directory}`, (err) => {
+    if (err) {
+      throw err;
+    }
+
+    console.log(`deleted ${directory} directory`);
+  });
+}
+
+function removeFilesOfType(type) {
+  const fileGlob = `packages/*!(node_modules|dist)/*.${type}`;
```

Comment:
I'm pretty sure the `!(...)` here is pointless because there is no `node_modules` or `dist` directly under `packages`. I get the same number of items out of this with or without it.